index.js

```diff
@@ -223,7 +227,42 @@ async function main (argv) {
   } catch (err) {
     process.stderr.write(`\nUnable to create socket files: ${err.stack ?? err.message}\n`)
   }
-  process.stdout.write('OK')
+  process.stdout.write('Ok.')
+
+  //
+  //  Initialize tsconfig.json when react_ts 
+  //
+  if (templateName === 'react-ts') {
+    try {
+      process.stdout.write('\nCreating tsconfig...')
+      await exec(
+        'npx tsc --init --declaration --allowJs --emitDeclarationOnly --jsx react-jsx --lib "dom","dom.iterable","esnext" --outDir dist'
```

Comment:
is it possible to enable strict mode by default?

Reply:
yes , we can do that by enable ``` --strict ``` flag

Reply:
yes, lets enable strict mode if that doesn't introduce typecheck or any other issues

Reply:
let me quickly test that

Reply:
It seems, that the latest versions of typescript are coming with strict mode by default set to true , so we don't need to provide  flag . There is a bunch of  additional options that developers can enable , the list below :

```
  /* Type Checking */
    "strict": true,                                      /* Enable all strict type-checking options. */
    // "noImplicitAny": true,                            /* Enable error reporting for expressions and declarations with an implied 'any' type. */
    // "strictNullChecks": true,                         /* When type checking, take into account 'null' and 'undefined'. */
    // "strictFunctionTypes": true,                      /* When assigning functions, check to ensure parameters and the return values are subtype-compatible. */
    // "strictBindCallApply": true,                      /* Check that the arguments for 'bind', 'call', and 'apply' methods match the original function. */
    // "strictPropertyInitialization": true,             /* Check for class properties that are declared but not set in the constructor. */
    // "noImplicitThis": true,                           /* Enable error reporting when 'this' is given the type 'any'. */
    // "useUnknownInCatchVariables": true,               /* Default catch clause variables as 'unknown' instead of 'any'. */
    // "alwaysStrict": true,                             /* Ensure 'use strict' is always emitted. */
    // "noUnusedLocals": true,                           /* Enable error reporting when local variables aren't read. */
    // "noUnusedParameters": true,                       /* Raise an error when a function parameter isn't read. */
    // "exactOptionalPropertyTypes": true,               /* Interpret optional property types as written, rather than adding 'undefined'. */
    // "noImplicitReturns": true,                        /* Enable error reporting for codepaths that do not explicitly return in a function. */
    // "noFallthroughCasesInSwitch": true,               /* Enable error reporting for fallthrough cases in switch statements. */
    // "noUncheckedIndexedAccess": true,                 /* Add 'undefined' to a type when accessed using an index. */
    // "noImplicitOverride": true,                       /* Ensure overriding members in derived classes are marked with an override modifier. */
    // "noPropertyAccessFromIndexSignature": true,       /* Enforces using indexed accessors for keys declared using an indexed type. */
    // "allowUnusedLabels": true,                        /* Disable error reporting for unused labels. */
    // "allowUnreachableCode": true,     
```
I was able to enable them all, after initializing react-ts template with no issue.  This is more advanced typescript, I think we should leave those options by default like that,  and who ever will need them, it'll uncomment desire one.  I'm thinking about ``` --alwaysStrict ``` flag that we can enable by default. Of course we can add few more flags  if you want ?

Reply:
I'm fine with the defaults 👍